Source/WebCore/testing/Internals.idl

```diff
@@ -958,7 +958,7 @@ typedef (FetchRequest or FetchResponse) FetchObject;
     readonly attribute unsigned long long storageAreaMapCount;
 
     unsigned long long elementIdentifier(Element element);
-    [CallWith=CurrentDocument] boolean isElementAlive(unsigned long long documentIdentifier);
+    boolean isElementAlive(unsigned long long documentIdentifier);
```

Comment:
I wonder what tests mean when they say “is alive”. Maybe we do want to pass the current document just so we can check the element is still in that document. One definition of alive is “still connected and still in this document”, which is what the old implementation required. But perhaps that was an accident and not important too preserve.

If the tests are intended to check the effects of garbage collection on object lifetimes, then it’s good that this functions semantics are based on `WeakHashMap` since that will tell if garbage collection has occurred and reclaimed the element’s memory. If not, then the fact that it’s sensitive to garbage collection’s effect on object lifetimes could actually make tests flaky if they are expecting elements to not be alive after a certain set of operations.

Now new: The argument name here is wrong. It’s the element identifier, not the document identifier.